anaconda.py

```diff
@@ -296,8 +296,8 @@ def start_debugger(signum, frame):
     # Set up logging as early as possible.
     from pyanaconda import anaconda_logging
     from pyanaconda import anaconda_loggers
-    anaconda_logging.init()
-    anaconda_logging.logger.setupVirtio()
+    anaconda_logging.init(not flags.imageInstall and not flags.dirInstall)
```

Comment:
Could you please make more obvious what the parameters means here? 
Maybe use keyword parameter or store this to a variable with name.

Reply:
Fixed.